.scripts/miniconda.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+echo "-- Installing latest Miniconda"
+if [ -d "$HOME/Deps/miniconda/bin" ]; then
+    echo "-- Miniconda latest version FOUND in cache"
+else
+    cd "$HOME"/Downloads
+    echo "-- Miniconda latest version NOT FOUND in cache"
+    if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+        curl -Ls https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh > miniconda.sh
+    elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
+        curl -Ls https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh > miniconda.sh
+    fi
+    # Travis creates the cached directories for us.
+    # This is problematic when wanting to install Anaconda for the first time...
+    rm -rf "$HOME"/Deps/miniconda
+    bash miniconda.sh -b -p "$HOME"/Deps/miniconda > /dev/null 2>&1
+    touch "$HOME"/Deps/miniconda/conda-meta/pinned
+    PATH=$HOME/Deps/miniconda/bin${PATH:+:$PATH}
+    hash -r
+    conda config --set show_channel_urls True > /dev/null 2>&1
+    conda config --set always_yes yes > /dev/null 2>&1
+    conda config --set changeps1 no > /dev/null 2>&1
+    conda update --all --yes > /dev/null 2>&1
+    conda clean -tipy > /dev/null 2>&1
+    conda info -a
+    cd "$TRAVIS_BUILD_DIR"
+    rm -f "$HOME"/Downloads/miniconda.sh
+fi
+echo "-- Done with latest Miniconda"
```

Comment:
has this done a bout with the latest molssi conda downloader to see who advances? https://github.com/MolSSI/QCFractal/blob/master/devtools/travis-ci/before_install.sh

Reply:
may as well, unless you see something that should be merged into it

Reply:
The cache check is missing from the MolSSI script. I'll merge the two.

Reply:
@dgasmith is that a MKL or no-MKL DL/install?

Reply:
oh, and when we switch to AM8 libint, that's also an issue (though probably could arrange AM6 for CI).

Reply:
MKL install which does take ~40 seconds on its own.

Reply:
Historically caching has provided an array of issues, but it might be worth seeing if the above causes any instabilities as these things do evolve quickly. So I would leave it in for now and simply be watchful.